arch/avr/src/at32uc3/at32uc3_lowconsole.c

```diff
@@ -348,11 +348,18 @@ void up_consoleinit(void)
 
 #endif
 
-  /* Enable selected clocks (and disabled unselected clocks) */
+  /* Enable selected clocks (and disable unselected clocks) */
 
   regval = getreg32(AVR32_PM_PBAMASK);
+#ifdef CONFIG_ARCH_CHIP_AT32UC3B
   regval &= ~(PM_PBAMASK_USART0 | PM_PBAMASK_USART1 | PM_PBAMASK_USART2);
   regval |= pbamask;
+#else
+
+  /* In the case of Mizar32, enable all USART clocks. */
+
+  regval |= PM_PBAMASK_USART0 | PM_PBAMASK_USART1 | PM_PBAMASK_USART2;
+#endif
```

Comment:
@ramangopalan UART console requires only a serial port enabled by time. The remaining serial ports could be enabled in other place for general purpose. I think you can keep the original code here.

Reply:
Dear @acassis, I will certainly check this tomorrow.